graylog2-server/src/main/java/org/graylog2/rest/models/system/sessions/responses/SessionValidationResponse.java

```diff
@@ -21,22 +21,39 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
+import javax.annotation.Nullable;
+
 @AutoValue
 @JsonAutoDetect
 public abstract class SessionValidationResponse {
     @JsonProperty("is_valid")
     public abstract boolean isValid();
 
+    @JsonProperty("new_session_id")
+    @Nullable
+    public abstract String newSessionId();
```

Comment:
The intention was that this field contains a different session id than the one which had been passed in. But I guess session_id is enough 👍